src/ansys/simai/core/data/projects.py

```diff
@@ -205,6 +205,14 @@ def compute_gc_formula(
 
         return gc_compute.result if gc_compute.is_ready else None
 
+    def cancel_training(self):
+        """Cancel training if there is one in progress."""
+
+        project_response = self._client._api.get_project(self.id)
+        if project_response.get("is_being_trained") is False:
+            raise ProcessingError("No training in progress for this project.")
```

Comment:
Why is it an error here but a no-op in ProjectDirectory ? Why is the return type different ?